backend/fms_core/viewsets/sample.py

```diff
@@ -62,6 +61,18 @@ class SampleViewSet(viewsets.ModelViewSet, TemplateActionsMixin, TemplatePrefill
             "template": [SAMPLE_SELECTION_QPCR_TEMPLATE["identity"]],
             "importer": SampleSelectionQPCRImporter,
         },
+        {
+            "name": "Prepare Libraries",
+            "description": "Upload the provided template with up to 384 new samples.",
```

Comment:
Maybe change the description to remove the false limit and better describe the template content.